contents/docs/tutorials/taxonomy-acquisition.md

```diff
@@ -1,36 +1,51 @@
 ---
-title: Setting up an Acquisition Taxonomy
+title: Set up Deep Dive Acquisition Dashboards
```

Comment:
Does this adhere to the "Use sentence case for titles" style guide rule? https://posthog.com/handbook/company/communication#writing-style

```suggestion
title: Set up deep dive acquisition dashboards
```

Reply:
Although the writing style guide could be a little clearer and be explicit about both page titles and page headings.